core/src/main/java/com/linecorp/armeria/common/Cookie.java

```diff
@@ -68,11 +68,25 @@ static Cookie of(String name, String value) {
      *
      * @param name the name of the {@link Cookie}
      * @param value the value of the {@link Cookie}
+     *
+     * @deprecated Use {@link #secureBuilder(String, String)} instead to create a secure {@link Cookie}.
      */
+    @Deprecated
     static CookieBuilder builder(String name, String value) {
         return new CookieBuilder(name, value);
     }
 
+    /**
+     * Returns a newly created {@link CookieBuilder} which builds a {@link Cookie} with secure settings.
+     * {@code "Secure"}, {@code "HttpOnly"}, and {@code "SameSite=Strict"} are enabled by default.
+     *
+     * @param name the name of the {@link Cookie}
+     * @param value the value of the {@link Cookie}
+     */
+    static CookieBuilder secureBuilder(String name, String value) {
```

Comment:
Let's also add
`static Cookie ofSecure(String name, String value) {...}`
This idea is from @jrhee17. 😄